tests/integration/any_charm.py

```diff
@@ -37,20 +38,29 @@ def _test_record_data(self) -> DNSRecordRequirerData:
         Returns:
             test record data
         """
-        dns_entry = RequirerEntry(
-            domain="dns.test",
-            host_label="admin",
-            ttl=600,
-            record_class="IN",
-            record_type="A",
-            record_data="42.42.42.42",
-            uuid=uuid.uuid4(),
-        )
-        dns_record_requirer_data = DNSRecordRequirerData(
-            dns_entries=[dns_entry],
-            service_account="fakeserviceaccount",
-        )
-        return dns_record_requirer_data
+        # We read the dns entries from a known json file
+        # It's okay to write to /tmp for these tests, so # nosec is used
+        with open("/tmp/dns_entries.json", "r", encoding="utf-8") as dns_entries_file:  # nosec
```

Comment:
I'm wondering if  `Path("/tmp/dns_entries.json").read_text(...)` here might be easier to read :D

Reply:
Agreed